pkgs/applications/misc/flashprint/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv
+, lib
+, fetchurl
+, dpkg
+, glibc
+, gcc-unwrapped
+, autoPatchelfHook
+, libGLU
+, qtbase
+, wrapQtAppsHook
+}:
+
+stdenv.mkDerivation {
+  name = "flashprint";
```

Comment:
Instead of `name`, `pname` should be used here
(if `name` is not set, it will be `"${pname}-${version}"` by default)